.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!--- Provide a general summary of your changes in the Title above -->
+<!--- Please follow this template to ensure you've satisfied all the requirements for having code merged, and to make the reviewer's life a bit easier -->
+
+## Related Issue
+
+<!--- Please link to the issue here: -->
+
+## Description
+
+<!--- Describe your changes in a bit more detail -->
+
+## How Has This Been Tested?
+
+<!--- Please describe in detail how you tested your changes. -->
+
+## Screenshots (if appropriate):
+
+<!--- Please provide before and after screenshots for any frontend changes -->
+
+## Checklist:
+
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. -->
+
+- [ ] My code follows the code style of this project. (see the [style guide](./docs/the-gazelle-style-guide.md))
```

Comment:
I'm not sure if this link will work within a PR template, didn't think I could test it, but just realized I can probably just try and edit this into my PR message and play with it to see if it works.